xesar-connect/src/main/kotlin/com/open200/xesar/connect/XesarConnect.kt

```diff
@@ -30,6 +29,7 @@ class XesarConnect(private val client: IXesarMqttClient, val config: Config) : A
     private val subscribedTopics = CopyOnWriteArraySet<String>()
     private val listeners = CopyOnWriteArrayList<Listener>()
     private val connectionChannel = Channel<ConnectionEvent>()
+    private val coroutineScopeForSendCommand = CoroutineScope(Dispatchers.IO)
```

Comment:
Maybe we could provide the coroutine scope optionally through the Config class. That could help with testing.

Reply:
Thx for the input :)  It will be done in a new issue because i need to change all tests for this change. Every test needs an own CoroutineScope instance.